src/js/components/devices/expanded-device.js

```diff
@@ -37,8 +41,41 @@ import MonitorDetailsDialog from './device-details/monitordetailsdialog';
 import DeviceNotifications from './device-details/notifications';
 import DeviceTwin from './device-details/devicetwin';
 
+const useStyles = makeStyles()(theme => ({
+  gatewayChip: {
+    color: theme.palette.grey[900],
+    path: {
+      fill: theme.palette.grey[900]
+    }
+  },
+  gatewayIcon: {
+    width: 20
+  }
+}));
+
 const refreshDeviceLength = 10000;
 
+const GatewayNotification = ({ device, docsVersion }) => {
+  const ipAddress = getDemoDeviceAddress([device]);
+  const { classes } = useStyles();
+  return (
+    <MenderTooltipClickable
+      placement="bottom"
+      title={
+        <div style={{ maxWidth: 350 }}>
+          For information about connecting other devices to this gateway, please refer to the{' '}
+          <a href={`https://docs.mender.io/${docsVersion}server-integration/mender-gateway`} target="_blank" rel="noopener noreferrer">
```

Comment:
We'll need to double check this once [MEN-5302](https://tracker.mender.io/browse/MEN-5302) is done - this specific part of how to connect a demo device to a demo Mender gateway will be covered on a get started guide, not on the more detailed docs under `server-integration`.

Reply:
Thanks, I marked the ticket as blocked by the other task... we could of course adjust this anytime afterwards too...